app/models/course/group.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Group < ActiveRecord::Base
+  stampable
+  belongs_to :course, inverse_of: :groups
+  has_many :group_users, inverse_of: :group, dependent: :destroy,
```

Comment:
@lowjoel Thanks ! It does not need the foreign_key here and I've removed it.  However, we need to specify the foreign_key in group_users side.